src/main/java/com/shapesecurity/salvation2/Policy.java

```diff
@@ -81,9 +80,8 @@ public static PolicyList parseSerializedCSPList(String serialized, PolicyListErr
 		List<Policy> policies = new ArrayList<>();
 
 		int[] index = { 0 }; // java's lambdas are dumb
-		PolicyErrorConsumer policyErrorConsumer = (Severity severity, String message, int directiveIndex, int valueIndex) -> {
+		PolicyErrorConsumer policyErrorConsumer = (Severity severity, String message, int directiveIndex, int valueIndex) ->
```

Comment:
I'm not a fan of this change; I generally use `{`-style to indicate that the return value of the expression isn't going to be used.

Reply:
Okay, I can revert those.